slides/k8s/accessinternal.md

```diff
@@ -0,0 +1,130 @@
+# Accessing internal services
+
+- When we are logged in on a cluster node, we can access internal services
+
+  (by virtue of the CNI model: all nodes can reach all pods and services)
```

Comment:
Do you think "CNI model" should be a link for further reading? It may not be super familiar to tutorial participants.

Reply:
Ah, this should have been "Kubernetes network model", not CNI model. Boo me.

And we describe the k8s network model in the beginning of the workshop.

I'll update that right away.